example/basic-redis.js

```diff
@@ -0,0 +1,30 @@
+import Redis from 'ioredis'
+import Fastify from 'fastify'
+import slowDownPlugin from '../index.js'
+
+let id = 0
+const redis = () =>
```

Comment:
nit: why a function? can't we just run it here?

Reply:
It's a function because we want to evaluate connection name for each server separately ;)

Reply:
ah ok understood